src/main/java/org/mitre/synthea/engine/Logic.java

```diff
@@ -170,7 +170,8 @@ public boolean test(Person person, long time) {
     private String operator;
     private List<Code> codes;
     private String referencedByAttribute;
-    private Double value;
+    private Object value;
```

Comment:
Just want to point out this change was more necessary than you realized because an Observation value could already be a String, not necessarily a Double.